### DISCUSSION TITLE
Strimzi with no ZK

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#6629] [Q&A]
I am only few months into Kafka and still learning.  I learned recently that   apache Kafka will stop using or supporting ZK as a component and all things which ZK did will be managed by Kafka cluster itself or something in that line! Is that correct and if it is , which strimzi version onwards can i test this.

**Comment by scholzj** [comment]
There is indeed work going on in Apache Kafka on removing the need for ZooKeeper. However, it is still work in progress with many different missing features. In Strimzi, we are making sure to be ready for when it is done, but there is currently no mode to try it because maintaining such feature while Kafka itself is not ready and is still missing many features would be a lot of effort to maintain it.
